LockdownFirewallWidget/LockdownFirewallWidget.swift

```diff
@@ -77,7 +77,7 @@ struct LockdownFirewallWidgetEntryView : View {
                 VStack(spacing: 0) {
                     Text(entry.dayMetricsString)
                         .font(.system(size: 21, weight: .semibold))
-                    Text("Blocked Today")
+                    Text(NSLocalizedString("Blocked Today", comment: ""))
```

Comment:
For consistency, maybe all NSLocalizedString keys should be uniform (capital/lowercase) going forward and then use the `uppercased()` function on String for styling.

Reply:
Confused as to what you mean - .uppercased would make the strings all caps: https://developer.apple.com/documentation/swift/string/1640996-uppercased

Can you clarify what you mean with an example? Thanks.

Reply:
Great question! I will happily clarify:
As seen in the `Localizable.strings` file, there are strings like (and including) ["DEACTIVATING" and "Deactivating..."](https://github.com/confirmedcode/Lockdown-iOS/blob/2989d914808a4777fc976b3d121ff3ac5d794a7a/Localizable.strings#L98). I am proposing instead that these kind of duplications become "Deactivating" = "Deactivating" (EN) where the usage can simply because `NSLocalizedString("Deactivating", comment: "").uppercased()` and `"\(NSLocalizedString("Deactivating", comment: ""))..."` respectfully.
Some other examples include "FIREWALL ON", "SHOW", "TUNNEL".
I think I may have confused you by accidentally commenting on line 80 instead of 67... sorry about that!

Reply:
Ah yes - that makes sense. Yes, I'll make a separate issue to deduplicate strings by case.

Reply:
#77